components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/dao/OrganizationManagementDAO.java

```diff
@@ -367,4 +367,15 @@ boolean isImmediateChildOfParent(String organizationId, String parentId) throws
      */
     int getRelativeDepthBetweenOrganizationsInSameBranch(String firstOrgId, String secondOrgId)
             throws OrganizationManagementServerException;
+
+    /**
+     * Get the parent organization id of the given organization and the depth.
+     *
+     * @param organizationId ID of the organization to get the parent.
+     * @param depth Depth of the parent organization to get.
+     * @return Parent organization id.
```

Comment:
comment needs to be updated properly 
parent is the immediate above level.
This method is to get any ansector of an org giving the difference of the depth

Reply:
Updated